CONTRIBUTING.md

```
@@ -0,0 +1,197 @@
+docfx ./docfx.json            # _site/ に出力
+docfx serve _site             # ローカルで閲覧
+```
+
+生成物（`_site/`・`api/`）はコミット対象です。GitHub Pages への反映は
+`.github/workflows/docs.yml` で自動化されています。
+
+## コミットメッセージ
+
+先頭に種別を表す接頭辞を付け、短い命令形で書きます。
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)